jwql/website/apps/jwql/views.py

```diff
@@ -309,7 +310,7 @@ def engineering_database(request, user):
         Outgoing response sent to the webpage
 
     """
-    edb_components = get_edb_components(request)
+    edb_components = get_edb_components(request, settings)
```

Comment:
Just curious: is there a reason why the settings need to be imported in `views.py` and passed into this function? Rather than just imported in `data_containers.py`?

Reply:
Probably a leftover from developing the code. I moved this as suggested.